tests/yast2_gui/yast2_bootloader.pm

```diff
@@ -53,9 +55,11 @@ sub run() {
     type_string 'dummy-password';
     send_key 'alt-y';
     type_string 'dummy-password';
+    wait_still_screen 3;
 
     # OK => Exit
     send_key "alt-o";
+    wait_still_screen 600;
```

Comment:
yes, but the first argument to `wait_still_screen` is the still time, not the timeout. See http://open.qa/api/testapi/#_wait_still_screen . Also that is unsafe because the screen might be still for quite some seconds but the creation of initrd is still not done. I suggest to use an additional `assert_screen 'generic-desktop', 600;` here